packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/execPermissionsWalkthrough.ts

```diff
@@ -3,12 +3,18 @@ import inquirer from 'inquirer';
 import path from 'path';
 import * as TransformPackage from 'graphql-transformer-core';
 import _ from 'lodash';
-import { topLevelCommentPrefix, topLevelCommentSuffix, envVarPrintoutPrefix } from '../../../constants';
+import { topLevelCommentPrefix, topLevelCommentSuffix, envVarPrintoutPrefix, CRUDOperation } from '../../../constants';
+import {
+  fetchPermissionCategories,
+  fetchPermissionResourcesForCategory,
+  fetchPermissionsForResourceInCategory,
+} from '../utils/permissionMapUtils';
+import { FunctionParameters, FunctionDependency } from 'amplify-function-plugin-interface/src';
 
 /**
  * This whole file desperately needs to be refactored
  */
```

Comment:
Can we remove this now 🤣 ?

Reply:
eh, I improved it, but it's by no means a finished product